gfo-light.ttl

```diff
@@ -0,0 +1,974 @@
+@prefix : <https://w3id.org/gfo/light/> .
```

Comment:
Why introducing the new RDF content in a different serialization (RDF/Turtle)? Existing RDF content is RDF/XML.